Source/ACE.Server/Factories/LootGenerationFactory.cs

```diff
@@ -35,5 +44,18050 @@ public static List<WorldObject> CreateRandomObjectsOfType(WeenieType type, int c
+                            //    }
+                            //    else if (chance == 110)
+                            //    {
+                            //        //Parabolic Ink
+                            //        id = 37356;
+                            //    }
+                            //    else if (chance == 111)
+                            //    {
+                            //        //Alacritous Ink
+                            //        id = 37359;
+                            //    }
+                            //    else if (chance == 112)
+                            //    {
+                            //        //Mana Scarab
+                            //        id = 37115;
+                            //    }
+                            //    else if (chance == 113)
+                            //    {
+                            //        //glyph of alchemy
+                            //        id = 37343;
+                            //    }
+                            //    else if (chance == 114)
+                            //    {
+                            //        //glyph of alchemy
+                            //        id = 37343;
+                            //    }
+                            //    else if (chance == 115)
+                            //    {
+                            //        //glyph of arcane lore
+                            //        id = 37344;
+                            //    }
+                            //    else if (chance == 116)
+                            //    {
+                            //        //glyph of armor
+                            //        id = 37345;
+                            //    }
+                            //    else if (chance == 117)
+                            //    {
+                            //        //glyph of armor tinkering
+                            //        id = 37346;
+                            //    }
+                            //    else if (chance == 118)
+                            //    {
+                            //        //glyph of bludgeoning
+                            //        id = 37347;
+                            //    }
+                            //    else if (chance == 119)
+                            //    {
+                            //        //glyph of cooking
+                            //        id = 37349;
+                            //    }
+                            //    else if (chance == 120)
+                            //    {
+                            //        //glyph of coordination
+                            //        id = 37350;
+                            //    }
+                            //    else if (chance == 121)
+                            //    {
+                            //        //glyph of corrosion
+                            //        id = 37342;
+                            //    }
+     
```

Comment:
These arrays will be created each time the method is called, and should ideally be marked as static

A list of ID numbers in the code should also eventually be a mapping to what these items actually are so they can be referenced easier